web/admin/src/App.js

```diff
@@ -246,6 +253,31 @@ export default class App extends Component {
     )
   }
 
+  parseDataForExport = () => {
+    let parsedData = []
+    const completed = this.state.attendees.filter(a => this.isDone(a.id))
+    this.state.attendees.forEach(attendee => {
+      if (this.state.allCodesByUser[attendee.id]) {
+        if (this.state.allCodesByUser[attendee.id].scans){
+          let gameWinningUser = completed.find(user => user.id === attendee.id)
+          let gameWinner = ""
+          gameWinningUser ? gameWinner = "True" : null
```

Comment:
Ternary operators shouldn't be used to *do* anything (like perform an assignment, aka `=`, operation); although this works, it is not idiomatic. 
 Replace this and the previous line with 
```suggestion
          const gameWinningUser = gameWinner ? "True" : null
```